pkg/pulsar/subscription.go

```diff
@@ -92,8 +95,14 @@ func (s *subscriptions) Create(topic utils.TopicName, sName string, messageID ut
 }
 
 func (s *subscriptions) Delete(topic utils.TopicName, sName string) error {
```

Comment:
IIUC pulsarctl doesn't intend to be used as a library, perhaps we simply add a bool flag `force` to `Delete`. Otherwise, `Delete` call `ForceDelete`, and `ForceDelete` has a bool flag `force` seems weird.

Reply:
Fixed.